Makefile

```diff
@@ -1,64 +1,56 @@
-SRCS := $(shell find . -name '*.go')
-PKGS := $(shell go list ./...)
-
-.PHONY: all
-all: deps lint test
+.PHONY: ci
+ci: deps lint test
 
 .PHONY: deps
 deps:
-	go get -d -v -t $(PKGS)
+	go get -d -v -t ./...
 
 .PHONY: updatedeps
 updatedeps:
-	go get -d -v -t -u -f $(PKGS)
+	go get -d -v -t -u -f ./...
 
 .PHONY: install
 install:
-	go install $(PKGS)
+	go install ./...
 
 .PHONY: golint
 golint:
 	@go get github.com/golang/lint/golint
-	for file in $(SRCS); do \
-		golint $${file}; \
-		if [ -n "$$(golint $${file})" ]; then \
-			exit 1; \
-		fi; \
-	done
+	golint -min_confidence 0.9 -set_exit_status ./...
```

Comment:
No need for double-running `go lint` if we use `-set_exit_status`. (Unfortunately, `gofmt` has no such flag.)

Also, the default confidence level of 0.8 caused it to complain about an exported method name. I don't like the idea of changing names in exported API just to satisfy a linter, so I up'ed this to make the warning go-away. (Unfortunately, it doesn't seem like this tool has other knobs for making exceptions and/or white-listing violations)